docs/source/en/_toctree.yml

```diff
@@ -326,6 +326,8 @@
         title: ESM
       - local: model_doc/falcon
         title: Falcon
+      - local: model_doc/fast
```

Comment:
this should be placed in the vision models section (or multimodal it that's applicable)